Gruntfile.js

```diff
@@ -281,6 +281,21 @@ module.exports = function (grunt) {
 					}
 				}
 			},
+			xhtml: {
```

Comment:
I think it would be easier to move this xhtml test into `tests/integrations`. That way we don't have to do all this scaffolding work. WDYT?

Reply:
these are not strictly speaking integration tests, but if you feel that will fit better there (and disturb less the code organization), sure! let me know.

Reply:
Yep, I think so.

Reply:
ok, I'll try to update the PR accordingly (I'm currently on vacation)